spec/controllers/application_controller_spec.rb

```diff
@@ -384,4 +384,202 @@
       expect(controller.moments_stats).to eq('<div class="center" id="stats">You have written a <strong>total</strong> of <strong>3</strong> moments. This <strong>month</strong> you wrote <strong>2</strong> moments.</div>')
     end
   end
+
+  describe 'generate_comment' do
```

Comment:
It would be worth running Rubocop over this file and just ignoring the "block too long" warnings